src/core/utils/error-chain.utils.ts

```
@@ -0,0 +1,97 @@
+    name: item.name,
+    severity: item.severity,
+    timestamp: item.timestamp.toISOString(),
+  }));
+}
+
+/**
+ * Returns a human-readable string representation of the error chain.
+ *
+ * @returns A string representation of the error chain.
+ */
+function toString(error: HandlerError): string {
```

Biome (1.9.4)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)